src/utilities/get-issue-numbers.ts

```diff
@@ -0,0 +1,23 @@
+export function getIssueNumbers(text: string): number[] {
+  let processedString = text;
+  if (processedString.includes('>')) {
+    processedString = processedString.replace(/>(?<=>)(.*)(?=\n)/g, '');
+  }
+  const issueStrings = processedString.match(/#[0-9]+/g);
+
+  const issueNumbers: number[] = [];
+
+  if (issueStrings) {
+    issueStrings.forEach((issue) => {
+      const issueNumberOrNaN = Number(issue.substr(1));
+
+      if (!isNaN(issueNumberOrNaN)) {
+        if (issueNumberOrNaN > 50) {
+          issueNumbers.push(issueNumberOrNaN);
+        }
+      }
+    });
+  }
```

Comment:
This can be changed to just:
```ts
if (!issueStrings) {
  // no matches
  return [];
}
return issueStrings
  // convert all strings to numbers, or NaN
  .map((issueString) => Number(issueString.substr(1)))
  // remove any non-numbers
  .filter((issueNum) => !isNaN(issueNum));
```

What cases could an `issueString` **not** be a number though? The regex should catch it right?

Also, I don't remember what the reasoning behind checking against `50` was.

Reply:
Checking against 50 was to avoid accidental misfires when I say something like "User Story #12 is asking you to centre your image"

Reply:
hmmm seems like another "magical feature/implicit feature"

Reply:
This `if` statement can be removed if the bot is only auto linking in the `#contributors` channel. What do you guys say?

Reply:
I'm on the side of **making this configurable** with a new environment variable for "issue_tag_channel", or similar. 

1. If the "issue_tag_channel" environment variable is passed, then the bot will listen for any issue numbers, including sub 50 numbers in that channel specified channel if it exists. 
2. If the "issue_tag_channel" environment variable is **not** passed, or the channel doesn't exist, then the bot will listen to all channels for any issue numbers.

This way we can easily tweak the behavior if we find the contributes only channel approach is to limiting, or if we find it more annoying. I think we can remove the "sub 50" limit since it seems to vague of a number to make much sense.